### ISSUE TITLE
Web address not working

### ISSUE THREAD

**Issue by Ben-BHNEL** [berkeley-homes/near-miss-positive-intervention#97]
@sohilpandya is the web address live as I cannot seem to access the page?

**Comment by sohilpandya** [comment]
Hi @Ben-BHNEL I've yet to transfer the domain across, I am still awaiting ownership of the account where we are hosting the app, I'll update you once that goes ahead.

**Comment by sohilpandya** [comment]
@Ben-BHNEL I've updated the settings and hopefully we should be able to access the app at http://goodmansfieldsnearmiss.co.uk/ within the next 24 hours. 

I'll update you on here once it is complete 👍

**Comment by sohilpandya** [comment]
Hi @Ben-BHNEL this should now be working fine. 👍

**Comment by Ben-BHNEL** [comment]
@sohilpandya when we submit a near miss we are not receiving an email. Can you please look into this

**Comment by des-des** [comment]
@Ben-BHNEL I am going to transfer the email now

**Comment by des-des** [comment]
@Ben-BHNEL, Okay, I have switched over the email now, you need to verify it. Do do this follow the link in the email received in goodmans.nearmiss@berkeleygroup.co.uk

**Comment by sohilpandya** [comment]
@Ben-BHNEL have you managed to verify the email? 

Are the emails coming through fine if you've verified?

**Comment by Ben-BHNEL** [comment]
@sohilpandya I am not receiving any emails from the app. Can you please look into this

**Comment by des-des** [comment]
@Ben-BHNEL I have sent a new conformation email to 	
goodmans.nearmiss@berkeleygroup.co.uk

Is this the correct email? You need to follow the link inside the email

**Comment by Ben-BHNEL** [comment]
@des-des I Haven't received this email as yet. Can you try this address. goodmansnearmiss@berkeleygroup.co.uk

**Comment by des-des** [comment]
@Ben-BHNEL Okay have done, If you cannot see it maybe check your junk also

**Comment by Ben-BHNEL** [comment]
@des-des Still not received an email for either address given and have checked the junk box also is this something you can look into?

**Comment by des-des** [comment]
@Ben-BHNEL is it possible for me to access that email address? On my end, it says the emails are going through..

**Comment by Ben-BHNEL** [comment]
@des-des Don't think you will be able to access our emails. Can we try my Berkeley email address which is ben.clarke@berkeleygroup.co.uk

**Comment by des-des** [comment]
@Ben-BHNEL We have something working on a new email.. @sohilpandya

**Comment by sohilpandya** [comment]
@Ben-BHNEL @Brian-GF I've sent an update over to you on friday, hope the new email address is getting all the form feedback emails 👍

**Comment by Ben-BHNEL** [comment]
@sohilpandya @Brian-GF Sorry for the delayed response. I am yet to receive any notifications of near misses via email. I have had IT look into it today and they have resolved a problem at our end. I have since tried recording a near miss via the app but haven't received an email notification.

**Comment by sohilpandya** [comment]
@Ben-BHNEL have you had a look at the email address that I've created and now redirecting all the responses to? 

We created a new email: goodmansfieldsnearmiss@gmail.com, this is where all the responses are being redirected to, as you can see in the screenshot below we are receiving the responses that are being sent. (just tested out a couple right now)
<img width="1678" alt="screen shot 2017-05-16 at 21 48 13" src="https://cloud.githubusercontent.com/assets/2305591/26127601/a3c7fbc6-3a81-11e7-9669-601d74140be0.png">


Also found the email for a response that was submitted by you at 5:06pm. (screenshot below) It seems to be working ok. 
<img width="1249" alt="screen shot 2017-05-16 at 21 50 43" src="https://cloud.githubusercontent.com/assets/2305591/26127643/d0af132c-3a81-11e7-84b2-72e36fec9188.png">

Please let me know if you are unable to find the email where I sent the details for accessing the new gmail account so that I can forward it to you again. 😊

**Comment by Ben-BHNEL** [comment]
@sohilpandya please can you forward the email for the gmail account as don't seem to have to the email

**Comment by sohilpandya** [comment]
@Ben-BHNEL have you been able to access the emails?

**Comment by Ben-BHNEL** [comment]
morning @sohilpandya  I am receiving emails now but we are having issues viewing photos. I have had IT look into this at our end and they are saying there is a problem with Amazon where the photos are stored. Can you look into this please

**Comment by sohilpandya** [comment]
Hi @Ben-BHNEL,

It might be better to talk about this over the phone so that we can go through it together, whens a good time for you to take a call?

**Comment by Ben-BHNEL** [comment]
@sohilpandya can you please change the thank you message once a report has been submitted to

" Thank you, your Near miss or Positive intervention has been recorded"

**Comment by sohilpandya** [comment]
@Ben-BHNEL Thanks for putting this on as a comment, I've created a new issue for this enhancement and will get back to you as soon as it has been implemented and live on the current site. 

https://github.com/berkeley-homes/near-miss-positive-intervention/issues/98
